liteflow-labs/liteflow-js
# Simplify example app

Fix https://github.com/liteflow-labs/liteflow-js/issues/127

Update the example app to use a simplified version of the login:
- Rainbow kit has been removed in favor of a plain wagmi connector
- Persistence has been removed, and no more localstorage/cookies as it brings a lot of complexity unnecessary for this app
- Improve how to test the current user
- Fix issue with changing account
- Separate offer and login as 2 pages to keep things simpler
- Update documentation

### How to test

You can just run the example locally and test to connect/disconnect/switch your wallet. You should always have a signature request to register yourself on the API
